pyron-plugin-impl/src/main/scala/com/cloudentity/pyron/plugin/impl/transformer/TransformRequestPlugin.scala

```diff
@@ -176,6 +196,7 @@ object TransformRequestOpenApiConverter extends OpenApiPluginUtils {
                 case BodyRef(_)      => // TODO implement
                 case PathParamRef(_) => // TODO implement
                 case HeaderRef(_, _) => // TODO implement
+                case _               => // TODO implement
```

Comment:
Let's remove '_' case - otherwise do not get non-exhaustive pattern match compile error if we add new ref type but we don't handle it here.

Reply:
ah, ok